war/src/main/js/keyboard-shortcuts.js

```diff
@@ -0,0 +1,26 @@
+import hotkeys from "hotkeys-js"
+
+window.addEventListener("load", () => {
+  const searchBar = document.querySelector("#search-box")
+  searchBar.placeholder = searchBar.placeholder + ` (${translateModifierKeysForUsersPlatform("CMD+K")
+    .replace("CMD", "⌘")})`
+
+  hotkeys(translateModifierKeysForUsersPlatform("CMD+K"), () => {
+    searchBar.focus()
+
+    // Returning false stops the event and prevents default browser events
+    return false
+  })
+})
+
+/**
+ * Given a keyboard shortcut, e.g. CMD+K, replace any included modifier keys for the user's
+ * platform e.g. output will be CMD+K for macOS/iOS, CTRL+K for Windows/Linux
+ * @param {string} keyboardShortcut The shortcut to translate
+ */
+function translateModifierKeysForUsersPlatform(keyboardShortcut) {
+  const useCmdKey = navigator.platform.toUpperCase().indexOf("MAC") >= 0 ||
+    navigator.platform.toUpperCase() === "IPHONE" ||
+    navigator.platform.toUpperCase() === "IPAD"
+  return keyboardShortcut.replaceAll(/CMD|CTRL/ig, useCmdKey ? "CMD" : "CTRL")
```

Comment:
Looks like `replaceAll` doesn't work in test cases, https://ci.jenkins.io/job/Core/job/jenkins/job/PR-6893/7/consoleFull

Reply:
I’ve hit that before you need to use replace I think

Reply:
> I’ve hit that before you need to use replace I think

According to ES, it's part of 21, if we pull https://github.com/jenkinsci/jenkins/pull/6887 and merge it into this PR we should be good to go.

Reply:
That didn't work out, does HtmlUnit not support ES6+ or anything modern released past 2015?

I guess we really need to use #replace again 😢

Reply:
ha i doubted this would work, no htmlunit is far behind in support and it's a nightmare, which is why moving ATH to here and ditching htmlunit would be great to do at some point, but yes for now it needs to be replace

Reply:
Fingers crossed it's sorted now 🤞

Reply:
> Fingers crossed it's sorted now 🤞

Build is green 👏🏻